journal/modernized/04-chapter-3.adoc

```diff
@@ -63,10 +64,15 @@ and at the close gave notice of a meeting at 4 o`'clock in the afternoon,
 to which Friends were requested to invite their neighbours; of whom a good many attended,
 and it ended to satisfaction.
 
-To His Wife.
+[.embedded-content-document.letter]
+--
 
+[.signed-section-context-open]
 Providence, Rhode Island, Eleventh Month 5th, 1794.
 
+[.letter-heading]
```

Comment:
Should the letter head and ssco be switched here?

Reply:
yep, the leader heading should be above the ssco. 👍